### PR TITLE
Sync with siddhi-5.0.0-m10 API changes

### PR DESCRIPTION
**Pull request by ajkarki**
## Purpose
Sync with siddhi-5.0.0-m10 API changes

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: docs/api/latest.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# API Docs - v4.1.1
+# API Docs - v5.0.0-SNAPSHOT
```
Comment (tishan89) [inline docs/api/latest.md]: Do not add this file

File: docs/api/5.0.0-SNAPSHOT.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
+# API Docs - v5.0.0-SNAPSHOT
```
Comment (tishan89) [inline docs/api/5.0.0-SNAPSHOT.md]: remove this file

File: docs/api/latest.md (hunk 1 above)
Comment (ajkarki) [reply docs/api/latest.md]: Fix with c9f4369

File: docs/api/5.0.0-SNAPSHOT.md (hunk 2 above)
Comment (ajkarki) [reply docs/api/5.0.0-SNAPSHOT.md]: Fix with c9f4369

**Pull request merged**